lib/evmone/eof.hpp

```diff
@@ -80,11 +91,11 @@ struct EOF1Header
     /// @data_size.
     uint16_t data_size = 0;
     /// Offset of data container section start.
-    uint16_t data_offset = 0;
+    uint32_t data_offset = 0;
     /// Size of every container section.
-    std::vector<uint16_t> container_sizes;
+    std::vector<uint32_t> container_sizes;
```

Comment:
Strictly speaking we don't need the changes here yet, given current code size limit.

But during header validation we do need to parse full 4-byte sizes and then use them to compare declared body size against actual body size, so we have there a vector of `uint32_t` container sizes already, and it's simpler and more efficient to just move it into this vector here.

Reply:
We actually should not copy the sizes to the separate vector but read these on demand. But this change is for some other time.